Dockerfile.tester

```diff
@@ -68,5 +68,7 @@ ENV SERVICE_PACKAGE=ibpc_tester
 ENV SERVICE_EXECUTABLE_NAME=ibpc_tester
 ENV SPLIT_TYPE=val
 
+COPY ibpc_tester/parts_in_search_phase1.json /opt/ros/overlay/install/lib/parts_in_search_phase1.json
```

Comment:
Let's have the ROS pkg install the file and retrieve the same from it's install dir.